.github/workflows/publish-csharp-apidocs.yml

```diff
@@ -26,11 +26,12 @@ jobs:
         clean: false
     - name: Move API docs into target area
       run: |
-        cd docs
-        cd api
-        mkdir csharp
-        cd ../../
-        MOVE csharp/ApiDocs/dest docs/api/csharp
+        rm -r -fo docs/api/csharp
```

Comment:
>rm -r -fo docs/api/csharp [](http://example.com/codeflow?start=8&length=25)

On windows rm is not there. Also CMD would not recognize forward slashes.

Reply:
So I had `del` for `cmd` but it didnt work in the gh action, I had to update to the powershell commands to get it to work.

Reply:
yea `fo` is `force` same as `f` I believe.